src/Datadog.Trace.ClrProfiler.Native/class_factory.cpp

```diff
@@ -45,7 +45,13 @@ HRESULT STDMETHODCALLTYPE ClassFactory::CreateInstance(IUnknown* pUnkOuter,
     return CLASS_E_NOAGGREGATION;
   }
 
-  trace::Info("Datadog CLR Profiler ", PROFILER_VERSION);
+  trace::Info("Datadog CLR Profiler ", PROFILER_VERSION,
+#ifdef BIT64
```

Comment:
Do we need different logic here for Windows vs Linux? Not sure if `BIT64` is defined for both.

Reply:
`BIT64` is defined for Linux in `src/Datadog.Trace.ClrProfiler.Native/CMakeLists.txt` and for Windows in `src/Datadog.Trace.ClrProfiler.Native/Datadog.Trace.ClrProfiler.Native.vcxproj`, so we should be good. I also tested this locally on Windows and observed it working